nginx.conf

```diff
@@ -13,13 +13,103 @@
 #
 # One difference between this file and the configuration for production is that
 # this configuration does not support request body decompression.
+daemon off;
+error_log stderr;
+pid ./tmp/nginx.pid;
 
 events {
 }
 http {
-  # This path may be different on your local computer depending on your OS and
-  # how you installed nginx.
-  include /usr/local/etc/nginx/mime.types;
+  access_log ./tmp/nginx-access.log;
+
+  types {
```

Comment:
The second column in this list is file extensions.  In the frontend `dist` directory, I can only find the following: `html ico js svg ttf woff`

We might get away with reducing the mime declarations to only these 6 extensions.

ref: https://nginx.org/en/docs/http/ngx_http_core_module.html#types

Reply:
I'd support limiting the size of this list. I also think it'd be nice if we rarely had to modify this list. What do you think about making this list those 6 extensions, then also adding extensions for a limited number of very common media types (whichever ones you think make the most sense)?

Reply:
Updated.

N.B. the following known-used types were missing:

* `ttf`
* `csv`

It might be worth checking the mime types that are currently being served for these in production.

Reply:
`ttf`: does not have it's own type - correctly served as `application/octet-stream`

Reply:
`csv`: this seems to be handled manually in `central-backend`: https://github.com/getodk/central-backend/blob/master/lib/resources/submissions.js#L307

Reply:
Thanks for checking those! :+1: